config/locales/en.yml

```diff
@@ -431,11 +438,22 @@ en:
               blank: "Enter their organisation"
             reason:
               blank: "Enter why they need access"
+        publish/user_form:
+          attributes:
+            email:
+              blank: "Enter an email address"
+              lowercase: "Enter an email address that is lowercase, like name@example.com"
```

Comment:
I don't think we need to validate whether an email is lowercase since `SimonWhatley@example.com` or `SIMONWHATLEY@EXAMPLE.COM` is a valid email. We might want to parse the email as lowercase when saving to the database.
```suggestion

```

Reply:
Yup done as mentioned above. 

Removed the translations here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/57a4e15e05361aa707ccda5fcbdc6ebfd2849e8e

We aren't using them anywhere else.